.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,59 @@
+name: "Docker build vX.Y.Z-clearmatics"
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+-clearmatics # Triggered by git tags like: v0.2.12-clearmatics
+
+jobs:
+  docker-build:
+    name: "Docker build and push"
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+
+    strategy:
+      matrix:
+        app: ['rnd']
```

Comment:
Out of curiosity, where is that used in the workflow?

Reply:
Don't you need to add `-${{ matrix.app }}` for the Github repo name?
i.e. `ghcr.io/$GITHUB_REPOSITORY-${{ matrix.app }}`

Reply:
Not need to add architecture to the tag even if you build images for multiple architectures. 

Matrix passed from template. Actually it does not need here. Removed.

Reply:
I see :)